spec/factories/course_levels.rb

```diff
@@ -0,0 +1,6 @@
+FactoryGirl.define do
+  factory :course_level, class: "Course::Level" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.